extensions/resource-deployment/src/interfaces.ts

```diff
@@ -204,7 +204,7 @@ export interface NotebookInfo {
 export enum OsType {
 	win32 = 'win32',
 	darwin = 'darwin',
-	linux = 'linux',
+	debian = 'debian',
```

Comment:
do we no longer support RHEL, Suse, etc. for this feature?

Reply:
we do, installation of those gets clubbed together with the default installation for all other linux distros. We will in future have native installers for rhel and suse when we will have separate OsType values and implementations for them.

Reply:
Kindly note, for all non-debian, non-ubuntu platforms the implementation does not change show we install azdata, we were using pip based installation before and we continue to do so.

Reply:
co-incidentally in a decision made after a review meeting on next steps for the auto-deployment, I will be pulling out the default pip3 based installation steps for azdata so we will no longer be supporting anything but debian for now (so no RHEL, no SUSE) until we get support of native installers from the platform team. This is because pip3 based installation has very poor customer experience and is hard to get right.